code/datums/traits.dm

```diff
@@ -296,6 +299,13 @@
 	onRemove(mob/owner)
 		owner.bioHolder?.RemoveEffect("deaf")
 
+TYPEINFO_NEW(/datum/trait/nolegs)
```

Comment:
You need both; `TYPEINFO` macro creates the new subtype, while `TYPEINFO_NEW` creates the `/New()` proc for that subtype
```suggestion
TYPEINFO(/datum/trait/nolegs)
TYPEINFO_NEW(/datum/trait/nolegs)
```

Reply:
Well that's annoying.

Reply:
I don't remember why this is the case but it's a thing, some Byond weirdness where it assumes you meant the existing type unless you specify.